test/middleware/spec.check-progress.js

```diff
@@ -1,7 +1,8 @@
 var checkProgress = require('../../lib/middleware/check-progress'),
     Model = require('../../lib/model'),
     Controller = require('../../lib/controller'),
-    helpers = require('../../lib/util/helpers');
+    helpers = require('../../lib/util/helpers'),
+    _ = require('underscore');
 
 describe('middleware/check-session', function () {
```

Comment:
This is an old mistake but should be 'middleware/check-progress'. I'll raise it as a separate PR.